### ISSUE TITLE
Reflect different transportation modes

### ISSUE THREAD

**Issue by perliedman** [perliedman/leaflet-routing-machine#130]
Use different line styles for different modes, indicate change of mode in itinerary.

https://github.com/Project-OSRM/osrm-backend/wiki/Server-api#response-2

**Comment by moimael** [comment]
Is there an option to specify transportation mode in LRM ? I haven't found it, and would be very interested using such thing :)

**Comment by perliedman** [comment]
@moimael no, currently not, but OSRM supports is as indicated in the link above. It should be fairly easy to add.

I've added this issue as a todo for the future.

**Comment by osro** [comment]
+1 to specify transportation modes.
